common/header.html

```diff
@@ -54,9 +54,23 @@
 
     html(attrs, state){
         let themeHtml = this.themeHtml(state.currentThemeId);
-        if (themeHtml) {
-          return [h('ul.menu-links.columned', themeHtml), h('.clearfix'),  h('hr')];
+        let sectionHeader = null;
+        const sectionHeaderText = "Theme Selector";
```

Comment:
Ideally this should be localised. There are some details about theme translations [here](https://meta.discourse.org/t/adding-localizable-strings-to-themes-and-theme-components/109867?u=david). You'll need to create the new directories/files in this theme, because it doesn't have any localised strings at the moment.